demos/fab.html

```diff
@@ -39,12 +43,57 @@
           right: 5rem;
         }
       }
+
+      fieldset legend {
+        display: block;
+        padding: 16px;
+        padding-top: 64px;
+        padding-bottom: 24px;
+      }
+
+      .hero button {
+        margin-left: 32px;
+        margin-right: 32px;
+
+      }
+
+      /* Override colors to match catalog theme. */
+
+      .mdc-fab {
+        background-color: #212121;
+      }
+
+      .mdc-fab--plain {
```

Comment:
Removed, may had been something I had to do when I had the CSS imports in the wrong order.